File: src/main/java/com/fengsheng/handler/skill_du_ji_c_tos.java
package com.fengsheng.handler;

import com.fengsheng.HumanPlayer;
import com.fengsheng.protos.Role;

public class skill_du_ji_c_tos extends AbstractProtoHandler<Role.skill_du_ji_c_tos> {
    @Override
    protected void handle0(HumanPlayer r, Role.skill_du_ji_c_tos pb) {
        r.getGame().tryContinueResolveProtocol(r, pb);
    }
}

File: src/main/kotlin/handler/skill_du_ji_c_tos.kt
package com.fengsheng.handler

import com.fengsheng.HumanPlayer
import com.fengsheng.protos.Role

class skill_du_ji_c_tos : AbstractProtoHandler<Role.skill_du_ji_c_tos>() {
    override fun handle0(r: HumanPlayer, pb: Role.skill_du_ji_c_tos) {
        r.game!!.tryContinueResolveProtocol(r, pb)
    }
}